client/src/components/ChannelInfo/index.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { Tabs, Tab, Typography, Box } from "@material-ui/core";
+import Cookies from "js-cookie";
+import axios from "axios";
+import { githubApiDomain } from "../../utils/constants";
+
+import "./index.css";
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`channelinfo-tabpanel-${index}`}
+      aria-labelledby={`channelinfo-tab-${index}`}
+      {...other}
+    >
+      <Box className="tabpanel-box">
+        <Typography component={"span"}>{children}</Typography>
+      </Box>
+    </div>
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.number.isRequired,
+  value: PropTypes.any.isRequired,
+};
+
+function a11yProps(index) {
+  return {
+    id: `channel-info-tab-${index}`,
+    "aria-controls": `channel-info-tabpanel-${index}`,
+  };
+}
+
+export default function ChannelInfo(props) {
+  const [repoInfo, setRepoInfo] = useState({});
+  const [isPrivate, setIsPrivate] = useState(false);
+
+  useEffect(() => {
+    const ghRepoInfo = async () => {
+      setIsPrivate(false);
+      try {
+        // Fetches repository information
+        const repository = props.location.pathname
+          .split("/")[2]
+          .replace("_", "/");
+        const headers = {
+          accept: "application/json",
+        };
+        if (Cookies.get("gh_private_repo_token")) {
+          headers["Authorization"] = `token ${Cookies.get(
+            "gh_private_repo_token"
+          )}`;
+        }
+        const ghRepoInfoResponse = await axios({
+          method: "get",
+          url: `${githubApiDomain}/repos/${repository}`,
+          headers: headers,
+        });
+        setRepoInfo(ghRepoInfoResponse.data);
+      } catch (error) {
+        console.log(error);
+        setIsPrivate(true);
+      }
+    };
+    ghRepoInfo();
+  }, [props.location.pathname]);
+
+  const [activeTab, setActiveTab] = useState(0);
+  const repoURL = `https://github.com/${props.location.pathname
+    .split("/")[2]
+    .replace("_", "/")}`;
+
+  const handleChange = (event, newValue) => {
+    setActiveTab(newValue);
+  };
+
+  return (
+    <div className="channel-info-wrapper">
+      <Tabs
+        value={activeTab}
+        onChange={handleChange}
+        aria-label="channel info tabs"
+        indicatorColor="none"
+      >
+        <Tab
+          className={`channel-info-tab ${activeTab == 0 ? "active-tab" : ""}`}
```

Comment:
Use === operator here as it is giving eslint warning

![image](https://user-images.githubusercontent.com/51796498/110286158-753ea480-800a-11eb-8b09-e8801bf0f237.png)

Same applies to line 98